### ISSUE TITLE
Cannot find module 'json5'

### ISSUE THREAD

**Issue by mudoo** [ChristianKohler/PathIntellisense#163]
```
workbench.desktop.main.js:sourcemap:3918 激活扩展“christian-kohler.path-intellisense”失败: Cannot find module 'json5'
Require stack:
- c:\Users\Administrator\.vscode\extensions\christian-kohler.path-intellisense-2.2.0\out\configuration\tsconfig.service.js
- c:\Users\Administrator\.vscode\extensions\christian-kohler.path-intellisense-2.2.0\out\configuration\configuration.service.js
- c:\Users\Administrator\.vscode\extensions\christian-kohler.path-intellisense-2.2.0\out\providers\javascript\javascript.provider.js
- c:\Users\Administrator\.vscode\extensions\christian-kohler.path-intellisense-2.2.0\out\providers\default.provider.js
- c:\Users\Administrator\.vscode\extensions\christian-kohler.path-intellisense-2.2.0\out\providers\index.js
- c:\Users\Administrator\.vscode\extensions\christian-kohler.path-intellisense-2.2.0\out\extension.js
- c:\Program Files\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Program Files\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Program Files\Microsoft VS Code\resources\app\out\bootstrap-fork.js。
_logMessageInConsole @ workbench.desktop.main.js:sourcemap:3918
_logOrShowMessage @ workbench.desktop.main.js:sourcemap:3919
$onExtensionActivationError @ workbench.desktop.main.js:sourcemap:4063
_doInvokeHandler @ workbench.desktop.main.js:sourcemap:4026
_invokeHandler @ workbench.desktop.main.js:sourcemap:4026
_receiveRequest @ workbench.desktop.main.js:sourcemap:4025
_receiveOneMessage @ workbench.desktop.main.js:sourcemap:4024
(anonymous) @ workbench.desktop.main.js:sourcemap:4022
fire @ workbench.desktop.main.js:sourcemap:95
fire @ workbench.desktop.main.js:sourcemap:424
_receiveMessage @ workbench.desktop.main.js:sourcemap:428
(anonymous) @ workbench.desktop.main.js:sourcemap:425
fire @ workbench.desktop.main.js:sourcemap:95
acceptChunk @ workbench.desktop.main.js:sourcemap:421
(anonymous) @ workbench.desktop.main.js:sourcemap:420
t @ workbench.desktop.main.js:sourcemap:431
emit @ events.js:203
addChunk @ _stream_readable.js:295
readableAddChunk @ _stream_readable.js:276
Readable.push @ _stream_readable.js:210
onStreamRead @ internal/stream_base_commons.js:166
workbench.desktop.main.js:sourcemap:3025 TypeError: Cannot read property 'filepath' of undefined
	at eval (.././src/core/KeyDetector.ts:95:57)
	at Array.find (<anonymous>)
	at Function.getUsages (.././src/core/KeyDetector.ts:95:45)
	at ReviewCommentProvider.provideCommentingRanges (.././src/editor/reviewComments.ts:123:74)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:632:240
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:51:988
	at new Promise (<anonymous>)
	at Object.t.asPromise (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:51:960)
	at e.$provideCommentingRanges (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:632:200)
	at e._doInvokeHandler (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:788:757)
	at e._invokeHandler (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:788:449)
	at e._receiveRequest (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:787:69)
	at e._receiveOneMessage (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:785:970)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:784:213
	at e.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:46:67)
	at v.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:232:615)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:957:179
	at e.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:46:67)
	at v.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:232:615)
	at t.PersistentProtocol._receiveMessage (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:237:17)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:234:155
	at e.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:46:67)
	at p.acceptChunk (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:230:129)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:229:483
	at Socket.t (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:238:565)
	at Socket.emit (events.js:203:13)
	at Socket.EventEmitter.emit (domain.js:476:20)
	at addChunk (_stream_readable.js:295:12)
	at readableAddChunk (_stream_readable.js:276:11)
	at Socket.Readable.push (_stream_readable.js:210:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:166:17)
workbench.desktop.main.js:sourcemap:1642   ERR Failed to execute git
```

**Comment by mudoo** [comment]
I don't know why no node_modules in the extension directory.

**Comment by valyakin** [comment]
Same, after upgrading to 2.2.0, when I open Extension Development Host in vscode

![image](https://user-images.githubusercontent.com/32761513/84259468-f4618380-ab31-11ea-8738-2eb441ffd7ea.png)

**Comment by macintacos** [comment]
Seeing the same thing in the extension logging on my end as well.

**Comment by ChristianKohler** [comment]
Thank you. Should be fixed with 2.2.1. Please check and let me know. Will close the issue as soon as I know it works.

**Comment by igorvolnyi** [comment]
The issue reappeared in 2.4.0
